man/docker-build.1.md

```diff
@@ -51,6 +52,24 @@ cloned locally and then sent as the context.
    the remote context. In all cases, the file must be within the build context.
    The default is *Dockerfile*.
 
+**--build-arg**=*variable*
```

Comment:
Since `docker-build.1.md` decribes the cli interface wecan't refer `JSON map...` part as here the environment is passed through flags.

Is following rewording good? We can remove the last paragraph if that seems too verbose. Let me know.

```
name and value of a **buildarg**.

For example, if you want to pass a value for `http_proxy`, use `--bulid-arg=http_proxy="http://some.proxy.url"`

Users pass these values at build-time. Docker uses the `buildargs` as the environment 
context for command(s) run via the Dockerfile's `RUN` instruction or for variable expansion in 
other Dockerfile instructions. This is not meant for passing secret values. [Read more about 
the buildargs instruction](/reference/builder/#arg)
```